VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -275,13 +275,12 @@ public void onSuccess(AuthResponse result) {
                             }
                         } catch (LoginException ex) {
                             super.onException(ex);
+                            onUserLoginFailure(ex, null, null);
```

Comment:
The LoginTask implementation has changed. It no longer throws an exception on a bad login. 

https://github.com/edx/edx-app-android/commit/e5d9b40f43f95e4b63f4bf7d4deaab99941afee5#diff-1c132d8bb33e47f173f915b58645cef7R28